test/test_es_utils.py

```diff
@@ -36,25 +37,37 @@ def test_get_bulk_uuids_embedded(es_client_fixture):
     uuids = ['1a12362f-4eb6-4a9c-8173-776667226988']  # only one uuid first
     result = get_bulk_uuids_embedded(es_client_fixture, 'fourfront-mastertestuser', uuids)
     assert len(result) == 1
+    assert result[0]['uuid'] == uuids[0]  # check uuid
+    assert result[0]['lab']['awards'] is not None  # check embedding
+    assert result[0]['lab']['awards'][0]['project'] is not None
+
 
     # one page of results, should give us 10
     users = execute_lucene_query_on_es(client=es_client_fixture, index='fourfront-mastertestuser', query={})
     uuids = [doc['_id'] for doc in users]
     result = get_bulk_uuids_embedded(es_client_fixture, 'fourfront-mastertestuser', uuids)
     assert len(result) == 10
+    for doc in result:
+        assert doc['uuid'] in uuids  # check uuids
 
 
 @pytest.mark.integrated
 def test_get_bulk_uuids_outperforms_expand_es_metadata(integrated_ff, es_client_fixture):
     """ Tests that the new method out performs the new one. """
     users = execute_lucene_query_on_es(client=es_client_fixture, index='fourfront-mastertestuser', query={})
     uuids = [doc['_id'] for doc in users]
+    current_time = [None]  # pass by reference
+
+    def set_current_time(start, end, t=current_time):  # noqa I want this default argument to be mutable
+        t[0] = end - start
 
```

Comment:
```
times = []
def set_current_time(start, end):
    times.append(end-start)
```
Then you don't have to do any copying between, can just call it twice, and can just make the second assertion do 
```
assert times[1] < times[0] / 2, "New functionality isn't twice as fast as clumsier technology."
```